docs/source/command_line.rst

```diff
@@ -322,6 +322,9 @@ Here are some more useful flags:
   annotations are not type checked.)  It will assume all arguments
   have type ``Any`` and always infer ``Any`` as the return type.
 
+- ``--disallow-incompletely-annotated-defs`` reports an error whenever it
```

Comment:
Yep, I agree! Flag is `--disallow-incomplete-defs` now.
I opted for the longer version originally because I wanted to avoid slight ambiguity in flag name. 

It's a tradeoff but I think shorter name is better.